app/views/users/index.html.haml

```diff
@@ -42,3 +47,33 @@
         %td= user.user_type.try(:humanize)
         %td= user.sid
         %td= link_to 'Edit', edit_user_path(user), :class => 'btn btn-primary'
+
+:javascript
+  $(() => {
+    var AUTH_TOKEN = $('meta[name="csrf-token"]').attr("content");
+    var html = `
+    <div id="PopoverContent">
+      <form action="users/import?&X-CSRF-Token=${AUTH_TOKEN}&X-Requested-With=XMLHttpRequest" enctype="multipart/form-data" method="post">
+        <div class="form-group">
+          <input type="file"
+          id="file" name="file" accept=".csv">
+        </div>
+        <div class="row" style="margin-left: 0px; align-items: center; ">
+          <button type="submit" class="btn btn-dark">Import</button>
+          <a href="#" class="tooltip-toggle" 
+            aria-label="Accepts comma delimited CSV file, with columns being of user_email,
+            user_name, org_name, etc..." 
+            tabindex="0" style="margin-left: 5px;">
+            Help
+          </a>
+        </div>
+      </form>
+    </div>`;
+    $('[data-toggle="popover"]').popover({
+      container: 'body',
+      html: true,
+      placement: 'bottom',
+      sanitize: false,
+      content: html
+    })
+  });
```

Comment:
This part is for a different feature: letting users upload a csv file to create new users